README.md

```diff
@@ -126,10 +126,12 @@ E.g. `foreman run rails server` or `foreman run rails console`.
     ``` user = User.last``` #or ```user = User.find_by(github_handle: "yourgithubhandle") ```
     ``` user.roles.create(name: "organizer") ```
     You can assign yourself other roles in the same way. If however you assign
-    yourself a student role AND another role, that may lead to unexpected behavior in the app. In that case, remove the student role.    
+    yourself a student role AND another role, that may lead to unexpected behavior in the app. In that case, remove the student role. 
```

Comment:
I wonder: do we still need to add the roles, like student, to ourselves? If not, we can delete the lines 128 + 129.